### PR TITLE
Implement MOTD command

### PR DESCRIPTION
**Pull request by A-UNDERSCORE-D**
Adds support for [/MOTD](https://modern.ircdocs.horse/#motd-message). This uses the config processing system added by @spb  in 8d37fdaf2dc7efe033e031cfda8a0fa96182e821

The reading of the config is done once at startup, rehashes are not included in this PR.
The infos config allows for infos to be omitted entirely, resulting in None. If an infos entry is included in the config, but _not_ a valid path on disk, sable will fail to start with a reasonable error

### CODE REVIEW

File: configs/server2_motd.txt
Comment (spb) [inline configs/server2_motd.txt:1]: If it's not referenced by the config, do we need this empty file?

File: sable_ircd/src/server/config.rs [hunk 1]
```diff
@@ -7,7 +10,63 @@ pub struct ListenerConfig {
...
+
 #[derive(Debug, Deserialize, Clone)]
 pub struct ClientServerConfig {
     pub listeners: Vec<ListenerConfig>,
+    #[serde(flatten)]
+    pub info_paths: InfoPaths,
+}
+
+#[derive(Serialize, Deserialize, Clone)]
+pub struct Infos {
```
Comment (spb) [inline sable_ircd/src/server/config.rs:26]: Can we give this a more descriptive name?

File: sable_ircd/src/server/config.rs [hunk 2]
```diff
@@ -7,7 +10,63 @@ pub struct ListenerConfig {
...
+    pub listeners: Vec<ListenerConfig>,
+    pub infos: Infos,
+}
+
+#[derive(Debug)]
+pub struct ConfigProcessingError {
+    reason: String,
+}
+
+impl std::error::Error for ConfigProcessingError {}
```
Comment (spb) [inline sable_ircd/src/server/config.rs:66]: Prefer deriving Error over implementing it by hand - use thiserror::Error and see the other examples in the crate

File: sable_ircd/src/server/mod.rs [hunk 3]
```diff
@@ -76,6 +79,9 @@ pub struct ClientServer {
 
     node: Arc<NetworkNode>,
     listeners: Movable<ListenerCollection>,
+
+    // Any general static info (responses for MOTD, ADMIN, and so on)
+    pub infos: Infos,
```
Comment (spb) [inline sable_ircd/src/server/mod.rs:84]: Same as above - a more descriptive name would be nice

**Review by spb — CHANGES_REQUESTED** [review]

File: sable_ircd/src/server/config.rs (hunk 2 above)
Comment (A-UNDERSCORE-D) [reply sable_ircd/src/server/config.rs:66]: Was looking for something like that, apparently I missed it! Thanks

File: configs/server2_motd.txt
Comment (A-UNDERSCORE-D) [reply configs/server2_motd.txt:1]: deleted

File: sable_ircd/src/server/mod.rs (hunk 3 above)
Comment (A-UNDERSCORE-D) [reply sable_ircd/src/server/mod.rs]: Per chat, now info_strings

File: sable_ircd/src/server/config.rs (hunk 2 above)
Comment (A-UNDERSCORE-D) [reply sable_ircd/src/server/config.rs]: and changed

**Pull request merged**